### DISCUSSION TITLE
more emojis

### DISCUSSION THREAD

**Discussion by xrtw** [discord-tickets/bot#113] [General]
hello, nice bot! i have one idea. if you click on other emoji, so other privileged people can react to member ticket

**Comment by eartharoid** [comment]
Glad you like the bot. 😄 

> if you click on other emoji, so other privileged people can react to member ticket

I don't understand what you mean by this. 🤔

**Reply by xrtw** [reply]
Sorry. I mean that it would be awesome if by clicking other emojis on ticket channel would add other members to responsive it. (Staff 1, Staff 2, Staff 3).. sorry

**Reply by eartharoid** [reply]
ok.
